Sources/Operators/RemoveKnownDuplicates.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  RemoveKnownDuplicates.swift
+//  CombineExt
+//
+//  Created by Jasdev Singh on 21/03/2020.
+//  Copyright © 2020 Combine Community. All rights reserved.
+//
+
+import Combine
+
+public extension Publisher where Output: Hashable {
+    /// When `Output` conforms to `Hashable`, `Publisher.removeKnownDuplicates` de-duplicates _all_
+    /// published value events, as opposed to pairwise with `Publisher.removeDuplicates`.
+    ///
+    /// - note: It’s important to note that this operator stores all emitted values
+    ///         in an in-memory `Set`. So, use this operator with caution, when handling publishers
+    ///         that emit a large number of unique value events.
+    ///
+    /// - returns: A type-erased publisher that consumes duplicate values across all previous emissions from upstream.
+    func removeKnownDuplicates() -> AnyPublisher<Output, Failure> {
+        scan((Set<Output>(), Output?.none)) { seenAndPrevious, incoming in
+            var seen = seenAndPrevious.0
```

Comment:
<img src="https://media.giphy.com/media/TanrwmKcRu7Yq577Fb/giphy.gif" />

Reply:
Best GIF (cc @krausefx)

Reply:
this is what i do when I can't find time to review
find a meme-able line of code, put a GIF, move on with my day 
💰

Reply:
Best move 😂